packages/lrod/lrod.install

```diff
@@ -0,0 +1,13 @@
+post_install() {
+  set -e
+  cd /usr/share/lrod
+  python -m venv venv
+  source venv/bin/activate &&
+    pip install --isolated --root="/usr/share/lrod" --prefix='venv' \
+      -r requirements.txt
+    pip install -e .
```

Comment:
```suggestion
```

there is no package to install, it's a standalone script

Reply:
Please test building and running your packages with ba-dev before submitting a PR